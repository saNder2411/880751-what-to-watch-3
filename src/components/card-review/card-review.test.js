import React from 'react';
import renderer from 'react-test-renderer';
import CardReview from './card-review';

const review = {
  id: 5,
  comment: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
  user: {
    id: 1,
    name: `Kate Muir`,
  },
  rating: 10,
  date: `2567`,
};

it(`Should CardReview render correctly`, () => {
  const markup = renderer
    .create(<CardReview {...review} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
