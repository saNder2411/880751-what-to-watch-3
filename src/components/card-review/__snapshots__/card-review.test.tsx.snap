// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CardReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kate Muir
      </cite>
      <time
        className="review__date"
        dateTime="2016-12-24"
      >
        January 1, 2567
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    10
  </div>
</div>
`;
